src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.iOS.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace System.Diagnostics
+{
+    public partial class Process
+    {
+        public TimeSpan PrivilegedProcessorTime => throw new PlatformNotSupportedException();
```

Comment:
If you instead add TargetsiOS condition here: https://github.com/dotnet/runtime/blob/91c1d5cc82061e4eb1bbd1cde52cb49ac05f33c0/src/libraries/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj#L355
you could avoid adding these new files. We throw PNSE on Unknown Unix from everything.

Reply:
Good to know, thanks!